main.py

```diff
@@ -1 +1,118 @@
 
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common.keys import Keys
+import time
+
+class Testcase101:
+
+    def main(self):
+        driver = webdriver.Firefox(executable_path="C:\\Users\\Johny\\Downloads\\geckodriver-v0.33.0-win64\\geckodriver.exe")
+        driver.get("https://interview.supporthive.com/staff/")
+        driver.implicitly_wait(30)
+        driver.maximize_window()
+        driver.find_element(By.ID, "id_username").send_keys("Agent")
+        driver.find_element(By.ID, "id_password").send_keys("Agent@123")
```

Comment:
Instead of hardcoding this kind of sensitive data it is prefered to use env variable.